client/state/ui/editor/image-editor/reducer.js

```diff
@@ -59,7 +60,12 @@ export function hasChanges( state = false, action ) {
 export function fileInfo( state = defaultFileInfo, action ) {
 	switch ( action.type ) {
 		case IMAGE_EDITOR_SET_FILE_INFO:
-			return Object.assign( {}, state, { src: action.src, fileName: action.fileName, mimeType: action.mimeType } );
+			return Object.assign( {}, state, {
```

Comment:
This is perfectly fine, but I couldn't help myself and toyed with some alternatives using destructuring and spread:

```js
const { src, fileName, mimeType, title } = action;
return { ...state, src, fileName, mimeType, title };
```

```js
const { type, ...info } = action;
switch ( type ) {
	case IMAGE_EDITOR_SET_FILE_INFO:
		return { ...state, ...info };
}
```